fusionfusion/seq_utils.py

```diff
@@ -7,9 +7,9 @@ def getSeq(reference, region_tuple):
 
     seq = ""    
     for reg in region_tuple_sorted:
-        for item in pysam.faidx(reference, reg[0] + ":" + str(reg[1]) + "-" + str(reg[2])):
-            if item[0] == ">": continue
-            seq = seq + item.rstrip('\n')
+        for line in pysam.faidx(reference, reg[0] + ":" + str(reg[1]) + "-" + str(reg[2]), split_lines = True):
```

Comment:
```suggestion
        for line in pysam.faidx(reference, reg[0] + ":" + str(reg[1]) + "-" + str(reg[2]), split_lines=True):
```

PEP8

> Don't use spaces around the = sign when used to indicate a keyword argument, or when used to indicate a default value for an unannotated function parameter:

https://www.python.org/dev/peps/pep-0008/#other-recommendations

Reply:
Thanks for the comment! Just fixed it.